package.json

```diff
@@ -20,7 +20,7 @@
     "core-js": "^2.5.3",
     "curl": "cujojs/curl#0.8.9",
     "domready": "^1.0.8",
-    "emotion": "^9.2.4",
+    "emotion": "^9.2.12",
```

Comment:
Where's this dependency come from? The audio player?

Reply:
@aware yes, this dependency is necessary to support the audio player. It’s a bit of a mess really.

Reply:
@aware yes, this dependency is necessary to support the audio player. It’s a bit of a mess really.